app/Community/Controllers/UnsubscribeController.php

```diff
@@ -18,8 +19,33 @@ public function __construct(
     ) {
     }
 
-    public function show(Request $request, string $token): InertiaResponse
+    public function show(Request $request, string $token): InertiaResponse|Response
     {
+        // Invalid signatures are handled automatically by the 'signed' middleware (returns 403).
+        // If we reach this point, the signature is valid.
+
+        // Handle POST requests for RFC 8058 one-click unsubscribe from email clients.
+        if ($request->isMethod('post')) {
```

Comment:
I can verify that the URL in the Unsubscribe header (as exposed by mailpit) does Unsubscribe the user, but I'm having trouble simulating the POST to actually exercise this bit of code. I'm going to assume it's working based on the unit tests.